pkg/rancher-prime/docs.ts

```diff
@@ -0,0 +1,109 @@
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/about-rancher/what-is-rancher.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config#external-authentication-configuration-and-principal-users',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/rancher-admin/users/authn-and-authz/authn-and-authz.html#_external_authentication_configuration_and_principal_users'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/admin-settings/authentication/google/#3-creating-service-account-credential',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/rancher-admin/users/authn-and-authz/configure-google-oauth.html#_3_creating_service_account_credentials'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/monitoring-alerting/configuration/#alertmanager-configuration/',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/observability/monitoring-and-dashboards/configuration/configuration.html#_alertmanager_configuration'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/rke1-vs-rke2-differences#cluster-api',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/cluster-deployment/rke1-vs-rke2.html#_cluster_api'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/amazon',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/set-up-cloud-providers/amazon.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/azure',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/set-up-cloud-providers/azure.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration#additionalmanifest',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/configuration/k3s.html#_additionalmanifest',
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/integrations-in-rancher/logging/logging-helm-chart-options',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/observability/logging/logging-helm-chart-options.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/monitoring-alerting/',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/observability/monitoring-and-dashboards/monitoring-and-dashboards.html',
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/faq/telemetry',
```

Comment:
this one i think was removed since the pr was created

Reply:
Removed